app/helpers/Emails/Notifications/SubmissionEmailsSender.php

```diff
@@ -37,7 +37,7 @@ class SubmissionEmailsSender {
   public function __construct(EmailHelper $emailHelper, array $params) {
     $this->emailHelper = $emailHelper;
     $this->sender = Arrays::get($params, ["emails", "from"], "noreply@recodex.mff.cuni.cz");
-    $this->submissionEvaluatedPrefix = Arrays::get($params, ["emails", "submissionEvaluatedPrefix"], "Submission Evaluated - ");
+    $this->submissionEvaluatedPrefix = Arrays::get($params, ["emails", "subject"], "Submission Evaluated - ");
```

Comment:
This feels like unintended change and victim of find-all replace-all and before that copy&paste :-)